src/main/java/ProductCalculatorService.java

```diff
@@ -1,66 +1,19 @@
-import java.time.LocalDate;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.stream.Collectors;
+import java.util.*;

```

Comment:
I think we should try to avoid *-imports if I remember correctly :)

Reply:
Done